roles/scalelite/defaults/main.yml

```diff
@@ -0,0 +1,207 @@
+---
+scalelite_docker_volumes_folder: /docker_volumes
+scalelite_docker_compose_folder: /scalelite
+scalelite_docker_compose_owner: root
+scalelite_docker_compose_group: root
+
+scalelite_certbot_image: certbot/certbot:v2.11.0
+# Custom UID to run certbot as
+scalelite_certbot_user_id: 9999
+scalelite_certbot_group_id: 9999
+# scalelite_certbot_terraform_telekom_os_access_key:
+# scalelite_certbot_terraform_telekom_os_secret_key:
+# scalelite_certbot_terraform_telekom_domain:
+# scalelite_certbot_terraform_telekom_tenant:
+# scalelite_certbot_terraform_telekom_dns_zone_id:
+# scalelite_certbot_terraform_state_bucket:
+# scalelite_certbot_terraform_state_key:
+# scalelite_certbot_terraform_state_region:
+# scalelite_certbot_terraform_state_endpoint:
+scalelite_certbot_terraform_version: 1.5.4
+scalelite_certbot_resources:
+  reservations:
+    cpus: '0.1'
+    memory: 10M
+  limits:
+    cpus: '0.3'
+    memory: 30M
+# scalelite_letsencrypt_email:
+
+
+
+scalelite_greenlight_enabled: false
+scalelite_greenlight_image_repository: greenlight
+scalelite_greenlight_image_tag: v3.1.0.2
+scalelite_greenlight_extra_instances: []
+scalelite_greenlight_default_locale: de
+scalelite_greenlight_log_level: warn
+scalelite_greenlight_allow_mail_notifications: true
+# scalelite_greenlight_smtp_server:
+# scalelite_greenlight_smtp_port:
+# scalelite_greenlight_smtp_domain:
+# scalelite_greenlight_smtp_username:
+# scalelite_greenlight_smtp_password:
+# scalelite_greenlight_smtp_auth:
+# scalelite_greenlight_smtp_sender
+# scalelite_bbb_tenant_api_key:
+scalelite_greenlight_smtp_auth: plain
+scalelite_greenlight_smtp_starttls_auto: true
+scalelite_greenlight_resources:
+  reservations:
+    cpus: '0.1'
+    memory: 150M
+  limits:
+    cpus: "{{ ansible_facts['processor_vcpus'] / 2 }}"
+    memory: "{{ ansible_facts['memory_mb']['real']['total'] / 2 }}M"
+
+# Postgres Docker image runs as postgres:postgres (70:70)
+scalelite_postgres_user_id: 70
+scalelite_postgres_group_id: 70
+scalelite_postgres_image: postgres:14.8-alpine
+scalelite_postgres_resources:
+  reservations:
+    cpus: '0.2'
+    memory: 50M
+  limits:
+    cpus: "{{ ansible_facts['processor_vcpus'] / 2 }}"
+    memory: "{{ ansible_facts['memory_mb']['real']['total'] / 2 }}M"
+
+scalelite_postgres_exporter_image: prometheuscommunity/postgres-exporter:v0.17.1
+scalelite_postgres_exporter_resources:
+  reservations:
+    cpus: '0.1'
+    memory: 10M
+  limits:
+    cpus: "0.5"
+    memory: "20M"
+
+scalelite_redis_image: redis:7.2.4-alpine
+# Redis Docker image runs as redis:redis (999:1000)
+scalelite_redis_user_id: 999
+scalelite_redis_group_id: 1000
```

Comment:
comment about different ids would be helpful

Reply:
See line 79